gpii/node_modules/flowManager/test/shared/BrowserChannelTestDefs.js

```diff
@@ -19,30 +19,38 @@ var fluid = require("infusion"),
 
 fluid.registerNamespace("gpii.tests.flowManager.browserChannel");
 
```

Comment:
OK, I'll stop belabouring the point, but these should use very simple test SR data and perhaps 1-2 settings per solution.  They'd be easier to read and we wouldn't be dragging around ghosts of whatever solutions/settings people choose to use for the test.  A good TODO that mentions getting rid of this in the LSR era would probably be sufficient.